### ISSUE TITLE
Formalize a backup and recovery procedure

### ISSUE THREAD

**Issue by malini** [thunderbird/appointment#607]
**Is your feature request related to a problem? Please describe.**
As we approach MVP, we must have adequate data recovery/handling in place.

**Describe the solution you'd like**
Let's scope out what we want to address and break up those tasks

I'd like to discuss the following at the minimum:
- set up db snapshots/recovery, and test recovery
- live replication/redundancy
- Add README.md file regarding our db infrastructure choices from above^ and any diagnostic tips if things go awry

**Comment by aatchison** [comment]
We are only using Neon, so doesn't that cover this? @davinotdavid 
I think it does, so we can close this

**Comment by davinotdavid** [comment]
Yep, Neon should already cover this. Ref: https://neon.com/docs/manage/backups.

Closing!
